virtcontainers/acrn.go

```diff
@@ -28,6 +30,25 @@ import (
 	"github.com/kata-containers/runtime/virtcontainers/utils"
 )
 
+// Since ACRN is using the store in a quite abnormal way, let's first draw it back from store to here
+
+// UUIDPathSuffix is the suffix used for uuid storage
+const (
+	UUIDPathSuffix = "uuid"
+	uuidFile       = "uuid.json"
+)
+
+// VMUUIDStoragePath is the uuid directory.
+// It will contain all uuid info used by guest vm.
+var VMUUIDStoragePath = func() string {
+	path := filepath.Join("/run/vc", UUIDPathSuffix)
```

Comment:
I think it would be better if this path were not hard-coded here - ideally `/var/vc/` would be defined in `pkg/katautils/config.go` and then passed to each hypervisor somehow.